cds_migrator_kit/videos/README.md

```diff
@@ -0,0 +1,47 @@
+# Weblectures Migration
+
+To run the migration of weblectures locally, follow these steps:
+
+## Dump a Subset of Records on Legacy
+
+Run the following command to dump a subset of records:
+
+.. code-block:: bash
+
+    inveniomigrator dump records -q '8567_x:"Absolute master path" 8567_d:/mnt/master_share* -980__C:MIGRATED -980__c:DELETED -5831_a:digitized' --file-prefix lectures --chunk-size=1000
+
+Place your dumps into the `cds_migrator_kit/videos/weblecture_migration/data/weblectures/dump/` folder, or update the `extract: / dirpath:` in `cds_migrator_kit/videos/weblecture_migration/streams.yaml`.
+
+For the files, modify the `migration_config.py` file located at `cds_migrator_kit/videos/migration_config.py`, specifically updating the following variables:
+
+- `MOUNTED_MEDIA_CEPH_PATH`
+
+## Media Path
```

Comment:
I guess we miss here on how to either mount ceph locally or how to copy over some of the files